SimpleConsoloGameApp/Starter.cs

```diff
@@ -0,0 +1,238 @@
+                    enemyWeapon = new Sword("Big Sword", "Sword", 5);
+                    return (enemy, enemyWeapon);
+                case 2:
+                    Console.WriteLine("You're enemy is Goblin");
+                    enemy = new Goblin("Goblin", "Warrior", 10, 1, 1, 3);
+                    enemyWeapon = new Sword("Short Sword", "Sword", 4);
+                    return (enemy, enemyWeapon);
+                default:
+                    return (null, null);
+            }
+        }
+
+        public IWeapon? CreateWeapon(ICharacter? character)
+        {
+            if (character.Specialization == "Wizard")
+            {
+                return new Wand("Elder Wand", "Wand", 1, 6);
+            }
+            else if (character.Specialization == "Warrior")
+            {
+                return new Sword("Long Sword", "Sword", 5);
+            }
+            else
+            {
+                return null;
+            }
+        }
+
+        public bool EnemyAction(ICharacter? enemyCharacter, ICharacter heroCharacter, IWeapon enemyWeapon, IWeapon heroWeapon, bool isBlock)
+        {
+            bool isHeroAlive = true;
+
+            if (isBlock == true)
+            {
+                int attackDamage = enemyWeapon.DealDamage() + enemyCharacter.Strength;
+                attackDamage = ((Sword)heroWeapon).BlockDamage(attackDamage);
+                return isHeroAlive = heroCharacter?.TakeDamage(attackDamage) ?? false;
+            }
+            else
+            {
+                int attackDamage = enemyWeapon.DealDamage() + enemyCharacter.Strength;
+                return isHeroAlive = heroCharacter?.TakeDamage(attackDamage) ?? false;
+            }
+
+            return isHeroAlive;
+        }
+
+        public (bool, bool) CharacterChoices(ICharacter? character, IWeapon characterWeapon, ICharacter? enemy)
+        {
+            bool ifEnemyAlive = true;
+            bool isBlock = true;
+
+            if (character?.Specialization == "Wizard")
+            {
+                Console.WriteLine("Do action: attack\n regen");
+                string? input = Console.ReadLine();
+                switch (input)
+                {
+                    case "attack":
+                        int attackDamage = characterWeapon.DealDamage() + character.Strength + ((Wizzard)character).ManaPoints;
+                        ((Wizzard)character).ManaPoints = ((Wizzard)character).MaxManaPoints - ((Wizzard)character).ManaPoints;
+                        ifEnemyAlive = enemy?.TakeDamage(attackDamage) ?? false;
+                        return (ifEnemyAlive, isBlock = false);
+                    case "regen":
+                        ((Wizzard)character).RegenerateManaPoints();
+                        ifEnemyAlive = true;
+                        return (ifEnemyAlive, isBlock = false);
+                }
+            } 
+            else if (character?.Specialization == "Warrior")
```

Comment:
You could use an actual type check (is operator) instead of some string value. Another option is to have an enum.